fmriprep/workflows/epi.py

```diff
@@ -157,8 +165,12 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
                                   ('outputnode.epi_mask_t1', 'epi_mask_t1')]),
         (discover_wf, func_reports_wf, [
             ('outputnode.acompcor_report', 'inputnode.acompcor_report'),
-            ('outputnode.tcompcor_report', 'inputnode.tcompcor_report')]),
-    ])
+            ('outputnode.tcompcor_report', 'inputnode.tcompcor_report'),
+            ('outputnode.ica_aroma_report', 'inputnode.ica_aroma_report')]),
+        (discover_wf, func_derivatives_wf, [
+             ('outputnode.aroma_noise_ics', 'inputnode.aroma_noise_ics'),
+             ('outputnode.melodic_mix', 'inputnode.melodic_mix')]),
+        ])
```

Comment:
Our current pattern is to pass anything that needs to go to derivatives to `outputnode`, and then pass the `outputnode` fields to `func_derivatives_wf.inputnode`. The reasoning is basically that if it's a derivative, it's something another workflow should reasonably expect access to.